src/Aquifer.API/Endpoints/Comments/Threads/Create/Endpoint.cs

```diff
@@ -34,10 +34,19 @@ private async Task<Response> CreateResourceContentVersionCommentThreadAsync(Requ
         var resourceContentVersion = await dbContext.ResourceContentVersions
             .AsTracking()
             .Include(x => x.CommentThreads)
+            .Include(x => x.ResourceContent)
             .SingleOrDefaultAsync(x => x.Id == req.TypeId, ct);
 
         EndpointHelpers.ThrowErrorIfNull<Request>(resourceContentVersion, x => x.TypeId, "No type found for given id.", 404);
 
+        List<ResourceContentStatus> notAllowedStatuses =
+            [ResourceContentStatus.TranslationAiDraftComplete, ResourceContentStatus.TranslationAwaitingAiDraft];
+
+        if (notAllowedStatuses.Contains(resourceContentVersion!.ResourceContent.Status))
+        {
+            ThrowError(x => x.TypeId, $"Resource for type {req.TypeId} is not in correct status");
+        }
+
```

Comment:
I'd like to put this in a Helper Method. Would Common be the best place to put it?